app.rb

```diff
@@ -16,31 +16,41 @@ def initialize
   end
 
   def list_people
-    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
+    if @people == []
+      puts
+      puts "No people in system."
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"}
+    end
   end
 
   def list_books
-    @books.each { |book| puts "Title: \"#{book.title}\" , Author: #{book.author}"}
+    if @books.length == 0
+      puts
+      puts "No books in system."
+    else
+      @books.each { |book| puts "Title: \"#{book.title}\" , Author: #{book.author}"}
+    end
```

Comment:
- [ ] Same issue here, I can create a book successfully but it is not being displayed. Kindly fix this issue 👍 
- Your console app:
![Annotation 2022-05-19 043918](https://user-images.githubusercontent.com/85113567/169200001-c6cbeee9-d427-49ea-bc07-8092f82e8c13.jpg)